src/readout/main.go

```diff
@@ -64,6 +64,12 @@ func pollSmartPi(config *smartpi.Config, device *i2c.Device) {
 	data := make([]float32, 22)
 	i := 0
 
+	// Pin the accumulator loop to full minutes.
+	startAt := time.Now().Truncate(time.Minute).Add(61 * time.Second)
+	<- time.After(time.Until(startAt))
```

Comment:
I'm not sure this is a good idea,  This means that every smartpi everywhere is going to be locked into reporting at at exactly the top of minute.  This would be very bad for any external service that might get messages from a number of smartpi devices.

There have been a number of articles out there about synchronized distributed timing being very bad for online services.